### PR TITLE
Fix test-sanity-docker failures

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/module_utils/network/vyos/config/ospf_interfaces/ospf_interfaces.py [hunk 1]
```diff
@@ -76,8 +76,12 @@ def generate_commands(self):
         """Generate configuration commands to send based on
         want, have and desired state.
         """
-        wantd = {entry["name"]: entry for entry in self.want}
-        haved = {entry["name"]: entry for entry in self.have}
+        wanted = {}
```
Comment (Qalthos) [inline plugins/module_utils/network/vyos/config/ospf_interfaces/ospf_interfaces.py:79]:
```suggestion
        wantd = {}
```

**Review by Qalthos — CHANGES_REQUESTED** [review]
This seems to be the issue

**Review by justjais — APPROVED** [review]

File: tests/sanity/ignore-2.10.txt [hunk 2]
```diff
@@ -1 +1,3 @@
 plugins/action/vyos.py action-plugin-docs # base class for deprecated network platform modules using `connection: local`
+plugins/module_utils/network/vyos/config/ospf_interfaces/ospf_interfaces.py compile-2.6!skip
```
Comment (Qalthos) [inline tests/sanity/ignore-2.10.txt:2]: What's the issue in this file? Are there other dict comprehensions that didn't get fixed?

**Review by Qalthos — COMMENTED** [review]
Looks good, but it seems odd that we're keeping up some 2.6 compatibility but still skipping the check

**Pull request merged**